eclair-core/src/test/scala/fr/acinq/eclair/router/FuzzyGraph.scala

```diff
@@ -0,0 +1,227 @@
+package fr.acinq.eclair.router
+
+import java.nio.file.{Files, Paths}
+import java.sql.DriverManager
+import de.heikoseeberger.akkahttpjson4s.Json4sSupport.ShouldWritePretty
+import fr.acinq.bitcoin.{ByteVector32, MilliSatoshi}
+import fr.acinq.bitcoin.Crypto.PublicKey
+import fr.acinq.eclair.ShortChannelId
+import fr.acinq.eclair.api._
+import fr.acinq.eclair.db.sqlite.SqliteNetworkDb
+import fr.acinq.eclair.router.Graph.GraphStructure.{DirectedGraph, GraphEdge}
+import fr.acinq.eclair.router.Graph.WeightRatios
+import fr.acinq.eclair.wire.{ChannelAnnouncement, ChannelUpdate, Color, NodeAddress}
+import org.json4s.{FileInput, jackson}
+import org.scalatest.FunSuite
+import scodec.bits.ByteVector
+import scala.util.{Failure, Random, Success}
+import scala.collection.immutable.TreeMap
+import scala.collection.mutable
+
+class FuzzyGraph extends FunSuite {
```

Comment:
This should be call `FuzzyGraphSpec` instead

Reply:
Done in 1e24241